### DISCUSSION TITLE
Preserving Spaces between paragraph.

### DISCUSSION THREAD

**Discussion by noorkaif97** [jsvine/pdfplumber#555] [Q&A]
Hi,

I am trying to extract a paragraph from a PDF but the extract_text() method is removing the spaces between the paragraph.
Example : 
The quick brown fox jumped over the lazy dog.

The quick brown fox.

The extract_text() method gives me the output as : 
The quick brown fox jumped over the lazy dog.
The quick brown fox.

I want to preserve the spaces between the lines.

Any suggestions on how to do this would be very helpful.

Thanks in advance

**Comment by samkit-jain** [comment]
Hi @noorkaif97 Appreciate your interest in the library. This is something that will be available in the upcoming v0.6.0 using the `layout=True` param in the `.extract_text(...)` method. See https://github.com/jsvine/pdfplumber/pull/532 for the details.
